cranelift/interpreter/src/step.rs

```diff
@@ -776,7 +776,30 @@ where
             arg(0)?,
             ValueConversionKind::RoundNearestEven(ctrl_ty),
         )?),
-        Opcode::Shuffle => unimplemented!("Shuffle"),
+        Opcode::Shuffle => {
+            if let InstructionData::Shuffle { mask, .. } = inst {
+                let mask = state
+                    .get_current_function()
+                    .dfg
+                    .immediates
+                    .get(mask)
+                    .unwrap()
+                    .as_slice();
```

Comment:
Can we move this into the `imm` function. It would be nice to centralize access to the immediates in that function